solutions/Day_1/Day_01.ipynb

```diff
@@ -0,0 +1,330 @@
+    "> between 2000 and 3200 (both included).The numbers obtained should be printed in a comma-separated sequence on a single line._**\n",
+    "\n",
+    "---\n",
+    "\n",
+    "### Hints:\n",
+    "\n",
+    "> **_Consider use range(#begin, #end) method._**\n",
+    "\n",
+    "---\n",
+    "\n",
+    "\n",
+    "\n",
+    "**Solutions:**\n",
+    "\n",
+    "- **Using for loops**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "cell_id": "7b7998d37eea45d588d6e0bc0a3f73d5",
+    "deepnote_cell_type": "code"
+   },
+   "outputs": [],
+   "source": [
+    "numbers = []\n",
+    "\n",
+    "for i in range(2000, 3001):\n",
+    "    if i % 7 == 0 and i % 5 != 0:\n",
+    "        numbers.append(str(i))\n",
+    "\n",
+    "print(*numbers, sep = ', ')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "cell_id": "1fe6f79e0a684cc6b4fd547b73fe00e0",
+    "deepnote_cell_type": "markdown"
+   },
+   "source": [
+    "---\n",
+    "\n",
+    "- **Using generators and list comprehension**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "cell_id": "9292136c54884d37bfa0ca3b96ac73e2",
+    "deepnote_cell_type": "code"
+   },
+   "outputs": [],
+   "source": [
+    "# create the generator expression\n",
+    "generator = (i for i in range(2000, 3001) if i % 7 == 0 and i % 5 != 0)\n",
+    "\n",
+    "# print the numbers as a comma-separated sequence\n",
+    "print(*generator, sep=',')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "cell_id": "5e7c290434134f6c88b96311a9df1417",
+    "deepnote_cell_type": "markdown"
+   },
+   "source": [
+    "# Question 2\n",
+    "\n",
+    "### **Question:**\n",
+    "\n",
+    "> **_Write a program which can compute the factorial of a given numbers.The results should be printed in a comma-separated sequence on a single line.Suppose the following input is supplied to the program: 8\n",
+    "> Then, the output should be:40320_**\n",
+    "\n",
+    "---\n",
+    "\n",
+    "### Hints:\n",
+    "\n",
+    "> **_In case of input data being supplied to the question, it should be assumed to be a console input._**\n",
+    "\n",
+    "---\n",
+    "\n",
+    "\n",
+    "\n",
+    "**Solutions:**\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "cell_id": "adf3564790b741a5aa2797a314db9b0d",
+    "deepnote_cell_type": "markdown"
+   },
+   "source": [
+    "- **Using While Loop**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "cell_id": "e8004dcbcc0d42a5932b85866ef4163a",
+    "deepnote_cell_type": "code"
+   },
+   "outputs": [],
+   "source": [
+    "n = int(input(\"Please enter the integer number: \"))  # input() function takes input as string type\n",
+    "\n",
+    "i = 1\n",
+    "factorial = 1\n",
+    "\n",
+    "while i<n+1:\n",
```

Comment:
Always surround these binary operators with a single space on either side: assignment (=), augmented assignment (+=, -= etc.), comparisons (==, <, >, !=, <>, <=, >=, in, not in, is, is not), Booleans (and, or, not).
 
`i<n+1` -> `i < n + 1` Here and below.

See style guide about statements https://peps.python.org/pep-0008/#other-recommendations